example/denoiser.js

```diff
@@ -0,0 +1,779 @@
+import {
```

Comment:
I think ideally this example would just include a single model to keep it focused and simple. An ideal model would include something with opacity and transmission so we can evaluate these kinds of problem cases. See the HDR example or DoF example. Though this is something that can be adjusted later if needed.

Reply:
I get that, and we can certainly limit the models. But I think it easier to switch to models that highlight the edge cases specifically vs trying to find a single model that fits every one.
LMK what you think and I'll adjust it

Reply:
I think just one model is good for now - for debugging we can swap between models to check issues. I think it's best to have a more simple example for users to reference.